app/src/lib/stores/app-store.ts

```diff
@@ -1826,6 +1826,36 @@ export class AppStore extends TypedBaseStore<IAppState> {
         this.statsStore.recordCoAuthoredCommit()
       }
 
+      const selectedState = this.getSelectedState()
+      if (
+        selectedState !== null &&
+        selectedState.type === SelectionType.Repository &&
+        selectedState.repository.gitHubRepository !== null
+      ) {
+        const { commitAuthor } = state
+        if (commitAuthor !== null) {
+          const account = getAccountForRepository(
+            this.accounts,
+            selectedState.repository
+          )
+          if (account !== null) {
+            const email = account.emails.find(
+              email =>
+                email.email.toLowerCase() === commitAuthor.email.toLowerCase()
+            )
+            if (email == null) {
+              this.statsStore.recordUnattributedCommit()
+            } else if (
+              account.endpoint === getEnterpriseAPIURL(account.endpoint)
```

Comment:
This isn't exactly right. The `getEnterpriseAPIURL` method will take any url, extract the hostname and tack on `/api/v3`. If you look at [getEnterpriseAccount](https://github.com/desktop/desktop/blob/9320f28eb602f9e61065c55642756eb2cf20888a/app/src/ui/app.tsx#L398-L405) in our App component you'll see that all it does it take the first account it can find that _isn't_ GitHub.com. In a similar vein we should flip the order of the conditions here and first check if `account.endpoint === getDotComAPIEndpoint()` and then add a blanket `else` if it doesn't and count that as being Enterprise.